azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -166,9 +167,12 @@ def start_extension_command(self, extension_name, command, cwd, env, stdout, std
                 def track_cgroups():
                     cgroups = self._cgroups_api.get_extension_cgroups(extension_name)
 
-                    # TODO: Add to tracking list
+                    for cgroup in cgroups:
```

Comment:
we also need to track the agent in create_agent_cgroups

Reply:
Good catch. I will add it here.